syz-fuzzer/fuzzer.go

```diff
@@ -187,10 +187,16 @@ func main() {
 		log.Fatalf("failed to connect to manager: %v ", err)
 	}
 	log.Logf(1, "connecting to manager...")
+	modules, err := host.CollectModulesInfo()
```

Comment:
Please move this chunk right after CollectMachineInfo, before `log.Logf(0, "dialing manager at %v", *flagManager)`.
Otherwise if it hangs/crashes in CollectModulesInfo, based on the log it will look like it hanged connecting to manager.